### ISSUE TITLE
[Proposal] Combining by asset group

### ISSUE THREAD

**Issue by spajz** [lightgear/asset#1]
Hi,
simple and nice package. I think it would be useful if it is possible to combine the files by group.
Example:

```
$asset->registerStyles($styles1, 'vendor/package', 'frontend');
$asset->registerStyles($styles2, 'vendor/package', 'frontend_ie');
...
{{ Asset::styles('frontend') }}
{{ Asset::styles('frontend_ie') }}
```

and then we get 2 combined css files called like the group

```
<link href="http://localhost/laravel/public/assets/styles/frontend.css">
<link href="http://localhost/laravel/public/assets/styles/frontend_ie.css">
```

Thanks

**Comment by ralf57** [comment]
Thanks for proposing.
I am refactoring and simplifying the package usage in the "develop" branch.
I'll see what I can do but no promises since I don't want to add more complexity.

**Comment by ralf57** [comment]
Closed by dc6be6181 (on develop branch)
